packages/node_modules/@ciscospark/widget-space/src/enhancers/errors.js

```diff
@@ -0,0 +1,95 @@
+import {connect} from 'react-redux';
+import {compose, lifecycle} from 'recompose';
+import {bindActionCreators} from 'redux';
+import {addError, removeError} from '@ciscospark/redux-module-errors';
+
+import messages from '../messages';
+
+export default compose(
+  connect(
+    (state) => state,
+    (dispatch) => bindActionCreators({
+      addError,
+      removeError
+    }, dispatch)
+  ),
+  lifecycle({
+    componentWillMount() {
```

Comment:
This is really cool

Reply:
Yea. This is just one of the utilities. There a a bunch of others that simply patterns we already use. It also has a few that allow us to take base presentational components and apply transforms to make them custom components that we can reuse easily